core/src/main/resources/org/springframework/security/core/ott/jdbc/one-time-tokens.sql

```diff
@@ -0,0 +1,6 @@
+create table one_time_tokens
```

Comment:
Can you please name this one-time-tokens-schema.sql to align with the current conventions in Spring Security?